src/include/stat.h

```diff
@@ -170,25 +172,26 @@ __wt_stats_clear(void *stats_arg, int slot)
             (void)__wt_atomic_addi64(&(stat)->fld, (int64_t)(value)); \
     } while (0)
 
-#define WT_STAT_DECRV(session, stats, fld, value)                                 \
-    do {                                                                          \
-        WT_STAT_DECRV_BASE(session, (stats)[(session)->stat_bucket], fld, value); \
+#define WT_STAT_DSRC_DECRV(session, stats, fld, value)                                 \
```

Comment:
Can I simplify the stats and remove this layer e.g, we instead have WT_STAT_DATA_DECRV call to straight to WT_STAT_DECRV_BASE, and delete WT_STAT_DSRC_DECRV (since they are all data source stats now)? 
I would do the same with WT_STAT_DSRC_INCR as well.

Reply:
@ruby-oujo I'm not sure I understand.  We have lots of code changed by this PR to call `WT_STAT_DSRC_INCRV`.  And that macro name would get deleted...?

Reply:
I agree with Will.  If we're getting rid of one or the other, we should keep `DSRC`.

Reply:
After trialing some changes, it turns out that we will need to keep both the macros because DSRC ones can pass the stats pointer while the DATA macros do not. This wasn't made clear with the naming, so I will change the naming so one will be WT_**STATP**_DSRC_DECR and the other WT_**STAT**_DSRC_DECR.